libs/backend/src/cast_vote_records/export.ts

```diff
@@ -423,6 +429,76 @@ async function exportSignatureFileToUsbDrive(
   return ok();
 }
 
+/**
+ * Updates the creation timestamp of a directory and its children files (assuming no
+ * sub-directories) using the one method guaranteed to work:
+ * ```
+ * cp -r <directory-path> <directory-path>-temp
+ * rm <directory-path>
+ * mv <directory-path>-temp <directory-path>
+ * ```
+ */
+async function updateCreationTimestampOfDirectoryAndChildrenFiles(
```

Comment:
I think there should be tests for this to ensure that the timestamps do actually change.

Reply:
👍 I was planning to add tests in subsequent PRs, but I can go ahead and add this one now since it's important

Reply:
https://github.com/votingworks/vxsuite/pull/3972/commits/d4ef8523f534eb0d41ffc4e2f83f4b14291f3c3c